Task_HelloWorld/index.html

```diff
@@ -1,15 +1,24 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Hello World Problem</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
-    </div>
-
-    <!-- Start coding below FOR HELLO WORLD TASK;  DO NOT TOUCH the code above -->
-  </body>
-</html>
```

Comment:
The text inside the paragraph (lines 15-21) contains HTML tags like <b>, <i>, <u>, and <br />. If you want to display those tags as text, you need to escape them using entities. For example, use '&lt;b&gt;' instead of '<b>'.